test/Http/ApiKeyPsr7RequestAuthorizerTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace CultuurNet\UDB3\Http;
+
+use GuzzleHttp\Psr7\Request;
+use ValueObjects\StringLiteral\StringLiteral;
+
+class ApiKeyPsr7RequestAuthorizerTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @test
+     */
+    public function it_can_authorize_requests()
```

Comment:
Test is in orde, maar de naamgeving kan misschien nog wat verbeterd worden. Bv `it_should_add_an_x_api_key_header()`, of `it_adds_an_x_api_key_header()` afhankelijk van welk formaat we nu eigenlijk prefereren ;)